README.md

```diff
@@ -1,46 +1,52 @@
-# unc0ver
+# Unc0ver
 ### The most advanced jailbreak tool
-![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
+![unc0ver logo](/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png)
 
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
 by [@pwn20wnd](https://twitter.com/Pwn20wnd) & [@sbingner](https://twitter.com/sbingner)<br/>
 UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](https://reddit.com/u/Samg_is_a_Ninja)<br/>
 
 ## The most outstanding changes over the other jailbreaks
-* All exploits in same app
-* Detailed error messages
+* All exploits in the same app
+* Detailed  error messages
 * Faster patches
 * More stable patches
 * No extra battery drain
 * No random freezes
-* No random slow downs
+* No random slowdowns
 * No data is logged or shared
 * No malware
 * Proper jailbreak state detection
 * Proper bootstrap extraction to fix issues such as Cydia not appearing after jailbreak
-* Native build of Cydia for iOS 11
+* Native and updated build of Cydia for iOS 11
 * Telesphoreo port for ARM64
-* Much faster Cydia
-* Much more stable Cydia
-* Much more modern looking and acting Cydia
 * Cydia skips uicache when not needed
 * Cydia supports iPhone X screen size
-* Cydia Substrate for tweak injection
-* Much faster ldrestart 
-* Much more stable ldrestart
-* Changes to Cydia were made with permission from Saurik 
+* Official Cydia Substrate for tweak injection
+* Much faster and more stable ldrestart 
+* Changes to Cydia made with permission from Saurik 
 * Option to skip loading daemons
 * Option to dump APTicket
 * Option to refresh icon cache
 * Option to disable auto updates
 * Option to block app revokes
 * Option to restore RootFS
-* Button to restart device
-* Button to open Cydia in case it doesn't appear on the Home Screen
+* Option to Export tfp0
+* Button to restart the device
+* Button to open Cydia if it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Built-In Update checker
 * Working debugserver
+* Detailed debugging 
+* Realtime Log window 
 * An awesome UI
-* Substrate and Cydia Support 
+
+## Currently supported exploits
+* VFS(Empty_list) by lan beer (11-11.3.1)
+* MultiPath by lan beer (11.2-11.3.1)
+* Async_Wake by lan beer (11.1.2)
+* Voucher Swap by Brandon Azad (11.2-12.1.2)
+* V1ntex by Tihmstar (11.2-11.4.1)
```

Comment:
* V3ntex by Tihmstar (12.0-12.1.2) partial support.

Reply:
will be added after full release or ETA sons will be hyped